File: Homework7/src/homework7/Bank.java

package homework7;
import java.util.ArrayList;

public class Bank {
    private String name;
    private ArrayList<BankAccount> accounts = new ArrayList<>();

    public Bank(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }
    
     public void almacenarCuentas( BankAccount bankAccount )
     {
         accounts.add(bankAccount);
         
     }

    public ArrayList<BankAccount> getAccounts() 
    {
        return accounts;
    }
    
    public void mostrarCuentas()
    {
        System.out.println("\n**CUENTAS**\n");
        for (int i= 0; i<accounts.size(); i++ )
        {
            System.out.println(i+": " + "Numero de cuenta: " +accounts.get(i).getAccountNumber() +" tipo de cuenta: " + accounts.get(i).getType());
        }
        
    }
    
    public void mostrarCuentaEnEspecifico(long accountNumber)
    {
       for (int i= 0; i<accounts.size(); i++ )
       {
           long account = accounts.get(i).getAccountNumber();
           if (account == accountNumber)
           {
               System.out.println(i+": " + "Numero de cuenta: " +accounts.get(i).getAccountNumber() +" tipo de cuenta: " + accounts.get(i).getType());
           }
       }
    }
    
}

File: bank.py
class Bank:
    __name = ""
    __accounts = []
    
    def __init__(self, name) :
        self.__name = name
        self.__accounts = [] 

    def get_name(self)  :
        return self.__name
    
    def almacenar_cuentas( self, account_number ) :
        self.__accounts.append(account_number)

    def get_accounts (self) :      
        return self.__accounts
    
    def mostrar_cuentas(self) :
        print("\n**CUENTAS**\n")
        for i in range (0, len (self.__accounts)):
            print("{} : Numero de cuenta: {} tipo de cuenta: {}".format(i,self.__accounts[i].get_account_number(),self.__accounts[i].get_type()))
        print("\n")
        
    def mostrar_cuenta_en_especifico(self,  account_number) :
    
       for i in range (0, len (self.__accounts)) :
           print( self.__accounts[i].get_account_number()) 
           account = self.__accounts[i].get_account_number()
           if int(account) == int(account_number) :
               print("{} : Numero de cuenta: {} tipo de cuenta: {}".format(i,self.__accounts[i].get_account_number(),self.__accounts[i].get_type()))
 
               